addons/im_livechat/__manifest__.py

```diff
@@ -97,6 +97,7 @@
         'im_livechat.assets_core': [
             'mail/static/src/core/common/**/*',
             'mail/static/src/utils/common/**/*',
+            'mail/static/src/discuss/core/common/discuss_core_common_service.js',
```

Comment:
You could pick everything in `discuss/**/common/**`, so we don't have to change in the near future :)

Reply:
@alexkuhn done!